lightning/lightning-client.md

```diff
@@ -192,7 +193,7 @@ $ lnd --version
 **Example** of expected output:
 
 ```
-> lnd version 0.16.3-beta commit=v$VERSION-beta
+> lnd version 0.16.4-beta commit=v$VERSION-beta
```

Comment:
Delete this change, comment: https://github.com/minibolt-guide/minibolt/pull/70/files#r1257216011